README.md

```diff
@@ -15,96 +15,48 @@ This project also uses [Node.js](https://nodejs.org), [Kettle](https://github.co
 ## How Do I Use It?
 
 ### Development
+* Server code can be found in `src/server`
+* Common JavaScript code can be found in `src/ui`, though a significant portion of the code may be present in the theme folders (see [Theme Customization](#Theme-Customization)).
+
+### Running the site
 * `npm install` to install dependencies
-* Run a CouchDB server on `localhost:5984` (Docker is an easy way to do this, we recommend the `apache/couchdb` image)
-* Run `server/src/js/db/dbSetup.js` to configure necessary CouchDB databases (this will also ensure your CouchDB instance is set up in [single-node mode](https://docs.couchdb.org/en/stable/setup/single-node.html))
+* Run a CouchDB server on `localhost:5984`
+    * Docker is an easy way to do this, we recommend the `apache/couchdb` image. To use that image via Docker, run this command: `docker run -p 5984:5984 -d apache/couchdb`
+* Run `node .\src\server\db\dbSetup.js` to configure necessary CouchDB databases (this will also ensure your CouchDB instance is set up in [single-node mode](https://docs.couchdb.org/en/stable/setup/single-node.html))
     * If you have an [admin user](https://docs.couchdb.org/en/stable/intro/security.html) configured on you CouchDB instance, you will need to provide the credentials to `dbSetup.js`. This can be done by setting the `COUCHDB_URL` environment variable. `dbSetup.js` will use the URL specified in `COUCHDB_URL` when connecting to CouchDB. For example: `COUCHDB_URL=http://admin:adminpassword@localhost:5984`
+* Create a `sjrk.storyTelling.server.config.json5` file in the style of `sjrk.storyTelling.server.config.json5.example`. This file configures the server and allows you to customize the current theme or configure a new one. Edit the values in the `globalConfig` section to do this. More on this in [Theme Customization](#Theme-Customization).
```

Comment:
It would be helpful to create a section to describe all properties and their purpose, that can specified in `globalConfig`.